resources/SettingsTabComponents/SettingsPane.qml

```diff
@@ -365,6 +365,8 @@ Rectangle {
             anchors.centerIn: parent
             anchors.verticalCenterOffset: 5
             onEditingFinished: {
+                if (settingType === "float" || settingType === "double")
+                    text = text.replace(",", ".");
```

Comment:
is this to support European style number delimiters? Is that what was the source of the original bug? I only skimmed the conversation in Jira.

Reply:
i'm not certain the bug (unable to change values in settings table) was due to the validators but this is just to support both `.` and `,`, how double can be inputted / ensure that its sending the correct setting writes to eliminate possibility for validator error(?) (this removes scientific notation as well, not sure how much that is used)